### PR TITLE
Updating RU translation

### PR DESCRIPTION
**Pull request by Orygeunik**
Hi!

It's me and new pack of russian words! :)

Source for update is commit 02c1ced9bf9aa44f82788775bab841929febfa08

Includes:
* Description into guidebook of Nuclear reactor;
* Main file with "main" words (eg name of items);
* A few actualizing of guidebook

Special thanks for the project and nice logic of nuclear reactor :)

### CODE REVIEW

**Comment by Orygeunik** [comment]
@Technici4n, see it, please

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json [hunk 1]
```diff
@@ -1,15 +1,79 @@
...
+    },
+    {
+      "type": "crafting",
+      "recipe": "modern_industrialization:electric_age/casing/nuclear_casing_asbl",
+      "text": "Основным компонентом является Ядерный Корпус, который изготовлен из ядерного сплава: смеси кадмия, бериллия и взрывоустойчивого сплава."
+    },
+    {
+      "type": "crafting",
+      "recipe": "modern_industrialization:electric_age/casing/nuclear_item_hatch_asbl",
+      "text": "Верхняя часть конструкции составляют люки для ядерных предметов или жидкости (или простые корпуса). Это входы и выходы реактора."
```
Comment (Gaider10) [inline src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:31]: Верхнюю

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json [hunk 2]
```diff
@@ -1,15 +1,79 @@
...
+      "type": "text",
+      "text": "Элементами активной зоны ядерного реактора являются нейтроны.$(br)Они производятся с помощью ядерного топлива.$(br2)Есть два типа нейтронов:$(li)быстрые;$(li)тепловые.$(br)Быстрые переносят энергию, а тепловые - нет.$(br)Нейтроны движутся по прямой линии, пока не встретят элемент или не выйдут из реактора (для быстрых нейтронов их энергия затем теряется)."
+    },
+    {
+      "type": "text",
+      "text": "Поток нейтронов отображается в графическом интерфейсе реактора.$(br2)Когда нейтрон встречает непустой люк, могут произойти две вещи: нейтрон поглощается или рассеивается.$(br)Рассеянный нейтрон случайным образом изменит направление.$(br2)Если рассеянный нейтрон быстрый, он может замедлиться, превратившись в тепловой."
+    },
+    {
+      "type": "text",
+      "text": "Все это приводит к передачи энергию от нейтрона к люку в виде тепла.$(br2)Поглощенный нейтрон останавливает своё движение и также передает свою энергию, как если бы это был быстрый нейтрон."
```
Comment (Gaider10) [inline src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:56]: энергии

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json [hunk 3]
```diff
@@ -1,15 +1,79 @@
...
-      "text": "Вы могли заметить, что ядерный реактор нельзя создать. Это просто потому, что он еще не закончен! Мы работаем над этим, наберитесь терпения. :-)$(br2)Надеемся, что вам понравился мод."
+      "text": "Каждый ядерный компонент имеет максимальное количество поглощений. При достижении предмет либо уничтожается, либо превращается в истощенную версию.$(br)Это особенно полезно для ядерного топлива, потому что некоторая часть U-238 превращается в плутоний в обедненной версии, что означает, что некоторая его часть может быть преобразована обратно в топливо."
     },
     {
       "type": "text",
-      "text": "Не волнуйтесь, мы запланировали гораздо больше контента - нам просто нужно время, чтобы все это написать! Если вы обнаружите ошибки или у вас есть интересные идеи, которые мы могли бы реализовать, дайте нам знать!$(br2)Вы можете найти наш сервер Discord на нашей странице Curseforge."
+      "text": "То же самое происходит с жидкостями: немного жидкости трансформируется после каждого поглощения нейтронов. Это может быть использовано для массового производства полезных изотопов, таких как дейтерий и тритий.$(br)В любом случаях результат отображается в REI."
```
Comment (Gaider10) [inline src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:76]: В обоих случаях

**Comment by Technici4n** [comment]
Oh right sorry, thanks for reminding me. Do you want to have a look at the review comments?

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json (hunk 1 above)
Comment (Orygeunik) [reply src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:31]: Fix

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json (hunk 2 above)
Comment (Orygeunik) [reply src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:56]: fix

File: src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json (hunk 3 above)
Comment (Orygeunik) [reply src/main/resources/data/modern_industrialization/patchouli_books/book/ru_ru/entries/midgame/nuclear_reactor.json:76]: fix

**Comment by Orygeunik** [comment]
> Oh right sorry, thanks for reminding me. Do you want to have a look at the review comments?

All good.
Comments accepted

**Comment by Technici4n** [comment]
Perfect, thanks a lot! :)

**Pull request merged**